.github/workflows/ci.yml

```diff
@@ -229,3 +229,20 @@ jobs:
       
       - name: Verify release artifact
         run: ./ci/verify_android_release.sh
+
+  docs:
+    name: Check for documentation errors
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Install rust toolchain
+        uses: dtolnay/rust-toolchain@nightly
```

Comment:
Question: Is using `nightly` intentional here when most of CI uses stable?

Reply:
I mainly took it for consistency with the rustls and webpki repos. 

We do that in those repos to be able to [use `doc_auto_cfg`](https://github.com/rustls/rustls/blob/fa605bfff02d7c2b7b31cca01ea2f5faf003fc3d/rustls/src/lib.rs#L337-L338). Probably we should use that here too. I will adjust.

Reply:
Add `doc_auto_cfg` in a separate commit.